src/Compilers/VisualBasic/Portable/Symbols/Source/ImplementsHelper.vb

```diff
@@ -351,6 +351,17 @@ DoneWithErrorReporting:
                         If Not errorReported Then
                             ' Further verification of found method.
                             foundMember = ValidateImplementedMember(implementingSym, foundMember, implementedMemberSyntax, binder, diagBag)
+
+                            If Not MembersHaveMatchingTupleNames(implementingSym, foundMember) Then
```

Comment:
This was marked as resolved, but there was no code changes. I'd like to follow up on this.

---
In reply to: [82475062](https://github.com/dotnet/roslyn/pull/14229#discussion_r82475062) [](ancestors = 82475062)